components/retro-go/targets/scratch-arcade/env.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+os.environ["IDF_TARGET"] = "esp32s3"
+os.environ["FW_FORMAT"] = "none"
```

Comment:
Can you update the format of your env.py to match the new format. It's very similar, see https://github.com/ducalex/retro-go/blob/master/components/retro-go/targets/odroid-go/env.py for reference.